src/pages-conditional/dapps.tsx

```diff
@@ -1661,7 +1658,11 @@ const DappsPage = ({
               <Column>
                 <H2>
                   <Translation id="page-dapps-metaverse-title" />{" "}
-                  <Emoji size={2} ml={"0.5rem"} text=":globe_with_meridians:" />
+                  <Emoji
+                    fontSize="5xl"
+                    ml={"0.5rem"}
```

Comment:
```suggestion
                    ml="2"
```
I think we can do the same change for ml here